### ISSUE TITLE
"sourcelink test" fails for 2 documents

### ISSUE THREAD

**Issue by atifaziz** [atifaziz/Hazz#12]
Running `sourcelink test` on [Fizzler.Systems.HtmlAgilityPack.1.2.1-ci-20200407t1951.symbols.nupkg](https://www.myget.org/feed/raboof/package/nugetsymbols/Fizzler.Systems.HtmlAgilityPack/1.2.1-ci-20200407t1951) ([also attached](https://github.com/atifaziz/Hazz/files/4449310/Fizzler.Systems.HtmlAgilityPack.1.2.1-ci-20200407t1951.symbols.zip)) for 2ca0a2ceed2146d0049d81654cd66ea75da2e607 produces errors:

```
1 Documents without URLs:
8edef9391a47680caec71f4ded6c435fd60d04067808138edf6d8fb16a81b307 sha256 csharp C:\Users\appveyor\AppData\Local\Temp\1\.NETStandard,Version=v1.3.AssemblyAttributes.cs
1 Documents with errors:
b76e2bda9a5853a1c1e97a6badca21a35a6039bb21e06cf89156470b5a02ee1d sha256 csharp C:\projects\hazz\src\obj\Release\netstandard1.3\Fizzler.Systems.HtmlAgilityPack.AssemblyInfo.cs
https://raw.githubusercontent.com/atifaziz/Hazz/2ca0a2ceed2146d0049d81654cd66ea75da2e607/src/obj/Release/netstandard1.3/Fizzler.Systems.HtmlAgilityPack.AssemblyInfo.cs
error: url failed NotFound: Not Found
sourcelink test failed
failed for lib/netstandard1.3/Fizzler.Systems.HtmlAgilityPack.pdb
1 Documents without URLs:
a6e03ae4df13fe05345e9022d1f1cd24ecae4bfd66db4843697c855d9f9335f4 sha256 csharp C:\Users\appveyor\AppData\Local\Temp\1\.NETStandard,Version=v2.0.AssemblyAttributes.cs
1 Documents with errors:
b76e2bda9a5853a1c1e97a6badca21a35a6039bb21e06cf89156470b5a02ee1d sha256 csharp C:\projects\hazz\src\obj\Release\netstandard2.0\Fizzler.Systems.HtmlAgilityPack.AssemblyInfo.cs
https://raw.githubusercontent.com/atifaziz/Hazz/2ca0a2ceed2146d0049d81654cd66ea75da2e607/src/obj/Release/netstandard2.0/Fizzler.Systems.HtmlAgilityPack.AssemblyInfo.cs
error: url failed NotFound: Not Found
sourcelink test failed
failed for lib/netstandard2.0/Fizzler.Systems.HtmlAgilityPack.pdb
2 files did not pass in dist\Fizzler.Systems.HtmlAgilityPack.1.2.1-ci-20200407t1951.symbols.nupkg
```

**Comment by atifaziz** [comment]
Apparently one of the .NET Core 3.1 SDK versions broke SourceLink; see shiftkey/dotnetcore-sourcelink-test-bug#1 for discussion and some of the gory details.

**Comment by atifaziz** [comment]
Closed via 2351c3c9cdd19b617de4fe7532b6580d225a87af that is merge of PR #13.
